lib/segment/src/types.rs

```diff
@@ -203,8 +203,29 @@ impl PartialEq for ScoredPoint {
 pub struct PointGroup {
     /// Scored points that have the same value of the group_by key
     pub hits: Vec<ScoredPoint>,
-    /// Value of the group_by key shared by all the hits
-    pub id: GroupId,
+    #[schemars(with = "GroupId")]
+    pub id: PseudoId,
```

Comment:
The fact you need `PseudoId` to lookup doesn't mean that `GroupId` become `PseudoId`. Especially if you need to hack JsonSchema for this

Reply:
Good point, I felt that making a new type would be redundant. But it may be the way to go